### PR TITLE
Check on Summary tab

### PR DESCRIPTION
**Pull request by thivya-deriv**
Card: https://app.clickup.com/t/20696747/BOT-1663
Description : Just a small check to ensure that the summary panel is working as expected when bot is run/stopped.

### CODE REVIEW

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

**Review by maliheh-deriv — APPROVED** [review]

File: cypress/support/commands/dBot.js [hunk 1]
```diff
@@ -28,3 +28,27 @@ Cypress.Commands.add('c_setBlockDuration', () => {
...
+Cypress.Commands.add('c_checkRunPanel', () => {
+  cy.findByText('Summary').click()
+  cy.get('#db-run-panel__clear-button').then(($clearButton) => {
+    if ($clearButton.is(':disabled')) {
+      // Check while bot is running
+      cy.findByTestId('dt_mock_summary_card').should('be.visible')
+      cy.findByRole('button', { name: 'Stop' }).should('be.visible')
+      cy.findByText('Total profit/loss')
+        .parent()
+        .find('[data-testid="dt_span"]')
```
Comment (abuzar-deriv) [inline cypress/support/commands/dBot.js:41]: Why not findByTestId()?

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]

File: cypress/support/commands/dBot.js (hunk 1 above)
Comment (thivya-deriv) [reply cypress/support/commands/dBot.js:41]: Abuzar the dt_span is being used in multiple places hence why i did not use that.

Comment (abuzar-deriv) [reply cypress/support/commands/dBot.js:41]: Doesn't matter. Once you chain with parent(), findByTestId() should find only that element.

**Review by meeyun-deriv — APPROVED** [review]
LGTM but please check Abuzar latest comment

**Pull request merged**